src/engraving/libmscore/slur.cpp

```diff
@@ -36,6 +36,7 @@
 #include "mscoreview.h"
 #include "navigate.h"
 #include "part.h"
+#include "rest.h"
```

Comment:
It looks like this #include is not used, is it?

Reply:
yup, forgot to remove that